templates/ep19/bs/talks/talk.html

```diff
@@ -19,25 +19,25 @@
 
 <div class="container page" id="talk_page">
     <nav aria-label="breadcrumb">
-        <ol class="breadcrumb">
-            <li class='breadcrumb-item'><a href="/">Home</a></li>
-            <li class='breadcrumb-item'><a href='#'>Talks</a></li>
-            <li class='breadcrumb-item active'>{{ talk_as_dict.title }}<li>
+        <ol class="breadcrumb">{# TODO: replace hardcoded paths by generated ones! #}
+            <li class="breadcrumb-item"><a href="/">Home</a></li>
+            <li class="breadcrumb-item"><a href="/events/sessions/#Talks">Talks</a></li>
+            <li class="breadcrumb-item active">{{ talk_as_dict.title }}<li>
         </ol>
     </nav>
 
     <div class="row">
         <div class="col-md-12">
-            <div class='talk'>
+            <div class="talk">
                 <h1>{{ talk.title }}</h1>
                 <h3>{{ talk.sub_title }}</h3>
                 <h5>{% for speaker in talk_as_dict.speakers %}
-                    {{ speaker.name }}{% if not forloop.last %}, {% endif %}
+                    <a href="{% url 'conference-speaker' slug=speaker.slug %}">{{ speaker.name }}</a>{% if not forloop.last %}, {% endif %}
```

Comment:
The `conference-speaker` is a deprecated view - this should be a url `'profiles:profile' profile_slug=speaker.slug`

The speakers should include a [`slug` key](https://github.com/EuroPython/epcon/blob/ep2019/conference/talks.py#L54). The slug is part of `AttendeeProfile` class.

Reply:
Good. That should be fine now.